File: Scheduler/Samples/Scheduler.MVC3/Controllers/QuickInfoController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

using SchedulerTest.Models;

using DHTMLX.Scheduler;
using DHTMLX.Scheduler.Data;
using DHTMLX.Common;


namespace SchedulerTest.Controllers
{
    public class QuickInfoController : Controller
    {
        public virtual ActionResult Index()
        {

            var sched = new DHXScheduler(this);

            sched.InitialDate = new DateTime(2011, 9, 19);

            sched.Data.Parse(new DHXSchedulerDataContext().Events);

            sched.Extensions.Add(SchedulerExtensions.Extension.QuickInfo);
            return View(sched);
        }

    }
}

File: Scheduler/Samples/Scheduler.MVC3.VB/Controllers/QuickInfoController.vb

Imports DHTMLX.Scheduler
Imports DHTMLX.Scheduler.Controls
Imports DHTMLX.Scheduler.Data

Namespace DHXSchedulerSamples
    Public Class QuickInfoController
        Inherits System.Web.Mvc.Controller

        '
        ' GET: /QuickInfo

        Function Index() As ActionResult
            Dim sched = New DHXScheduler(Me)

            sched.InitialDate = New DateTime(2011, 9, 19)
            sched.Data.Parse((New DHXSchedulerDataContext()).Events)

            sched.Extensions.Add(SchedulerExtensions.Extension.QuickInfo)

            Return View(sched)
        End Function

    End Class
End Namespace